src/commonmark/en/content/user/data-visualizer.md

```diff
@@ -2,54 +2,323 @@
 
 ![](resources/images/data-visualizer/data-visualizer-dimensions.png)
 
-You can also change the selections by clicking on the
-chips in the layout area.
+Just like in the dimensions menu, in the layout area you can also change the selections by clicking on a dimension, dragging and dropping a dimension or by using a dimension's context menu (three dots).
 
 ![](resources/images/data-visualizer/data-visualizer-layout-area.png)
 
+- **Series**: A series is a set of continuous, related elements (for
+  example periods or data elements) which you want to visualize in
+  order to emphasize trends or relations in its data. Also known as Columns for Pivot table visualizations.
+
+<!-- end list -->
+
+- **Categories**: A category is a set of elements (for example indicators
+  or organisation units) for which you want to compare its data. Also known as Rows for Pivot table visualizations.
+
+<!-- end list -->
+
+- **Filter**: The filter selection will filter the data displayed in the
+  visualization. Note that if you use the data dimension as filter, you can
+  only specify a single indicator or data set as filter item, whereas
+  with other dimension types you can select any number of items.
+
+### Select dimension items
+
+<!--DHIS2-SECTION-ID:data_vis_select_dim_items-->
+
+A dimension refers to the elements which describe the data values in the system. There are three main dimensions in the system:
+
+- **Data**: Includes data elements, indicators and datasets (reporting
+  rates), describing the phenomena or event of the data.
+
+<!-- end list -->
+
+- **Periods**: Describes when the event took place.
+
+<!-- end list -->
+
+- **Organisation units**: Describes where the event took place.
+
+Data Visualizer lets you use these dimensions completely flexible in terms of appearing as series, categories and filter.
+
+To select items for each one, open the dimension modal window by clicking on a dimension. This window will also be opened automatically when adding a dimension with out items to the layout. Select which items to add to the visualization by double-clicking an item or by selecting an item with a single click and using the arrows in the middle. The order of appearance will be the same as the order in which they are selected. Selected items can be reordered by dragging and dropping them in the Selected section.
+
+![](resources/images/data-visualizer/data-visualizer-dimension-modal.png)
+
+#### Select periods
+
+When selecting a Period you have to option to choose between fixed periods and relative periods. These can also be combined. Overlapping periods are filtered so that they only appear once. For relative periods the names are relative to the current date, e.g. if the current month is March and **Last month** is selected, the month of February is added.
+
+![](resources/images/data-visualizer/data-visualizer-period-dimension-modal.png)
+
+#### Select organisation units
+
+TODO: NEEDS INFO
```

Comment:
The Classic docs has this part: https://github.com/dhis2/dhis2-docs/blob/DV-add-from-classic-DV/src/commonmark/en/content/user/classic-data-visualizer.md#select-organisation-units, however I believe most of this has changed with the current org unit selector. I'm not fully aware of how this works, maybe @janhenrikoverland or @edoardo could help me here?